src/client/utils/throttle.js

```diff
@@ -0,0 +1,31 @@
+import nativeMethods from '../sandbox/native-methods';
+
+export default function throttle (func, ms) {
+    var isSuspend = false;
```

Comment:
The task is more complicated.
We need an addition cookie sync for the following cases (if it is not done yet):
1)XmlHttpRequest and Fetch requests (store in custom header)
2)Images, script and style loading (pass via additional url parameter)
3)Page unload (store into custom input)

Waiting for https://github.com/DevExpress/testcafe-hammerhead/pull/721